### ISSUE TITLE
Should the "8" on this line be "dim_vector" ?

### ISSUE THREAD

**Issue by sinjax** [XifengGuo/CapsNet-Keras#2]
https://github.com/XifengGuo/CapsNet-Keras/blob/843a765f1242a7dbe46511ef5ad4f1501e6c9a46/capsulelayers.py#L171

**Comment by XifengGuo** [comment]
Yes, it is. Thanks for pointing out. I'll update the code very soon. @sinjax
